### PR TITLE
Add a parameter for the panic buffer size

### PR DESCRIPTION
**Pull request by nigoroll**
64k ought to be enough for everyone. /s

### CODE REVIEW

**Review by dridi — APPROVED** [review]
I would change the maximum from 1GB to 1MB, because 1GB seems excessive.

File: include/tbl/params.h [hunk 1]
```diff
@@ -732,6 +732,18 @@ PARAM_SIMPLE(
...
 
+PARAM_SIMPLE(
+	/* name */	panic_buffer,
+	/* type */	bytes_u,
+	/* min */	"4k",
+	/* max */	"1g",
+	/* def */	"64k",
+	/* units */	"bytes",
+	/* descr */
+	"Size of the panic message buffer.",
```
Comment (asadsa92) [inline include/tbl/params.h:743]: Maybe add a note here that enough space where shared memory is stored is required.
I am not sure about the wording, but a small note here would help.

**Review by asadsa92 — APPROVED** [review]
LGTM, minor comment below

**Comment by nigoroll** [comment]
@dridi 

> I would change the maximum from 1GB to 1MB, because 1GB seems excessive.

You mean, 1MB ought to be enough for everyone?

I would prefer a limit which is waaaay out. At first I thought it could be 1TB, but we _really_ should have a _lot_ of headroom.

@asadsa92 thank you for your suggestions, do you see them reflected in the SQUASHME commit?

**Review by asadsa92 — APPROVED** [review]
Thanks, the docs looks good now.

**Comment by bsdphk** [comment]
Ok with me.

**Comment by dridi** [comment]
> You mean, 1MB ought to be enough for everyone?

YES!

I could be convinced to go one order of magnitude higher, but 10MB sure ought to really be enough for everyone. You can fit a couple holy books in 10MB, and I don't expect anyone to read through a book-worth of panic output.

**Pull request merged**